### ISSUE TITLE
focus on and open toggle menu 

### ISSUE THREAD

**Issue by Billito** [onokumus/metismenu#31]
Hello,

I want to focus on and open toggle menu item.
I mean for example if I click on first level nav I want to show a page and to open the accordion menu to see the second level nav. Right now it's only possible to open the accordion menu.
Is that possible?

**Comment by ChaseGiunta** [comment]
I'm am also looking for this solution.@billito did you ever figure something out?

**Comment by onokumus** [comment]
Yes, but these are two separate behaviors. MetisMenu can open the submenu, while navigation to another page is handled by the link itself. Since navigating to another page reloads the document, the parent menu must be rendered as active on the destination page:

```html
<li class="mm-active">
  <a href="/example-page" aria-expanded="true">My Pages</a>
  <ul class="mm-collapse mm-show">
    ...
  </ul>
</li>
```

If the first-level item should only toggle the submenu, use `href="#"` or a separate toggle control. If it should both navigate and toggle, the application must handle that behavior; metisMenu does not determine the active page from the URL automatically.
